src/Fitting/Stats.jl

```diff
@@ -0,0 +1,115 @@
+# =========================== src/Fitting/Stats.jl ===========================
+# Conventions:
+# - Rows = variables (p), columns = samples (n)
+# - This file exposes the StatsBase/Distributions interface for CopulaModel
+
+# ---------- 1) Utils ----------
+@inline _as_pxn(C::Copulas.Copula, U::AbstractMatrix) = (size(U,1) == length(C)) ? U : permutedims(U)
+@inline _as_pxn(p::Integer,         U::AbstractMatrix) = (size(U,1) == p)        ? U : permutedims(U)
```

Comment:
Please STOP with this function ^^ 

The standard is that data HAS to be passed in (d,n) size, point. If the user pass it as (n,d) then screw them they get a n-dimensional copula, we do not help with it.